### DISCUSSION TITLE
Only the Beta works for me but if I change/add resolution ...

### DISCUSSION THREAD

**Discussion by frviana** [VirtualDrivers/Virtual-Display-Driver#144] [General]
I've tried the 3 drivers (non HDR, HDR and Beta)
For some reason only the Beta works for me. The other two I get a yellow exclamation mark and error 31.
The beta works fine but has limited resolution on the XML.
I tried add or change the XML for additional resolution (ipad resolution) but the Display options don't show the new resolution.

**Comment by bud3699** [comment]
This sounds more like an issue

And in this case Id only the beta works. That most likely means the configurations files are in the wrong folder. Therefore the beta is falling back to its pre-built  configurations. 

Make sure the files are in the correct folder corresponding to the instructions. The beta release also supports changing directory locations by a registry key if this may be needed

**Reply by BSteezie** [reply]
I'm having a similar issue (Windows 10), with the latest non-beta release (23.12.2.1)...the virtual display just didn't show in Display Settings.   However, the latest beta (24.10.27) installed fine and showed in Display Settings. But there are a limited number of resolutions in the xml file, and adding a resolution to the xml file seems to have no effect.   I need to add 2732x2048.  Is there a trick to adding a resolution, or is it just not referencing vdd_settings.xml in the C:\IddSampleDriver folder?

**Reply by bud3699** [reply]
> I'm having a similar issue (Windows 10), with the latest non-beta release (23.12.2.1)...the virtual display just didn't show in Display Settings. However, the latest beta (24.10.27) installed fine and showed in Display Settings. But there are a limited number of resolutions in the xml file, and adding a resolution to the xml file seems to have no effect. I need to add 2732x2048. Is there a trick to adding a resolution, or is it just not referencing vdd_settings.xml in the C:\IddSampleDriver folder?

You need to 1) make sure the xml is located in the `C:/VirtualDisplayDriver` folder. 2) Update resolutions and restart the driver or restart device for the xml to take effect

**Comment by tildebyte** [comment]
>  2) Update resolutions and restart the driver or restart device for the xml to take effect

This *does not work* with the installer. The XML file is *definitely* in the path pointed to by the `VDDPATH` registry key, and AFAICT, I haven't janked up the syntax.

Everything else is working fine - I installed the driver manually, the virtual display shows up, it's usable via e.g. Sunshine/Moonlight... changing the configuration via XML DOES NOT WORK; settings reloading/driver reloading/driver un-re- install/Win rebooting... nothing works.

I have no further info to give you, as even turning on logging is impossible (changing the config via the systray Companion does nothing as well).

**Reply by bud3699** [reply]
You can force logging via the registry, but that sounds like issue reading the path from registry, can you provide exact system information such as windows version, exact windows version

**Reply by tildebyte** [reply]
Windows 11 Pro, version 24H2, build 26100.3476

**Reply by tildebyte** [reply]
### Registry  entries

Under 'Computer\HKEY_LOCAL_MACHINE\SOFTWARE\MikeTheTech\VirtualDisplayDriver'

```
LOGS: REG_DWORD 0x00000001 (1)
VDDPATH: REG_SZ "E:\Programs\_System tools\_Display tools\VirtualDisplayDriver"
```

VDD_PATH contains e.g. "MttVDD.dll", "nefconw.exe", "unins000.exe", and of course "vdd_settings.xml"
